src/plugins/dashboard/public/services/dashboard_content_management/lib/check_for_duplicate_dashboard_title.ts

```diff
@@ -44,21 +50,38 @@ export async function checkForDuplicateDashboardTitle(
     return true;
   }
 
+  const [baseDashboardName, duplicationId] = extractTitleAndCount(title);
+
   const { hits } = await contentManagement.client.search<
     DashboardCrudTypes['SearchIn'],
     DashboardCrudTypes['SearchOut']
   >({
     contentTypeId: DASHBOARD_CONTENT_ID,
     query: {
-      text: title ? `${title}*` : undefined,
-      limit: 10,
+      text: title ? `${baseDashboardName}*` : undefined,
```

Comment:
Instead of falling back to undefined here, can we just cancel the whole operation if the `title` is undefined?

Reply:
It shouldn't be undefined, but it's possible to receive an empty string.  We can certainly return early and skip the whole check.